tests/jvm/src/test/scala/scala/meta/tests/semanticdb/SemanticdbSuite.scala

```diff
@@ -114,28 +134,35 @@ abstract class SemanticdbSuite extends FunSuite
     }
   }
 
+  def javaSymbols(code: String, expected: String): Unit = {
+    checkSection(code, expected, "Symbols", s.Language.JAVA)
+  }
+
   def occurrences(code: String, expected: String): Unit = {
-    checkSection(code, expected, "Occurrences")
+    checkSection(code, expected, "Occurrences", s.Language.SCALA)
   }
 
   def diagnostics(code: String, expected: String): Unit = {
-    checkSection(code, expected, "Diagnostics")
+    checkSection(code, expected, "Diagnostics", s.Language.SCALA)
   }
 
   def symbols(code: String, expected: String): Unit = {
-    checkSection(code, expected, "Symbols")
+    checkSection(code, expected, "Symbols", s.Language.SCALA)
   }
 
   def synthetics(code: String, expected: String): Unit = {
-    checkSection(code, expected, "Synthetics")
+    checkSection(code, expected, "Synthetics", s.Language.SCALA)
   }
 
-  private def computeDatabaseAndOccurrencesFromMarkup(markup: String): (s.TextDocument, List[String]) = {
+  private def computeDatabaseAndOccurrencesFromMarkup(
+      markup: String,
+      language: s.Language = s.Language.SCALA
```

Comment:
Can we drop the default value here as well?

Reply:
Done.